source/localizable/models/customizing-serializers.md

```diff
@@ -652,25 +652,54 @@ If you would like to create a custom serializer its recommend that you
 start with the `JSONAPISerializer` or `JSONSerializer` and extend one of
 those to match your needs. However, if your payload is extremely
 different from one of these serializers you can create your own by
-extending the `DS.Serializer` base class. There are 3 methods that
-must be implemented on a serializer.
-
-- [normalizeResponse](http://emberjs.com/api/data/classes/DS.Serializer.html#method_normalizeResponse)
-- [serialize](http://emberjs.com/api/data/classes/DS.Serializer.html#method_serialize)
-- [normalize](http://emberjs.com/api/data/classes/DS.Serializer.html#method_normalize)
-
-Its also important to know about the `normalized` JSON form that Ember
-Data expects as an argument to `store.push()`.
-
-`store.push` accepts a JSON API document. However, unlike the
-JSONAPISerializer, `store.push` does not do any transformation of the
-record's type name or attributes. It is important to make sure that
-the type name matches the name of the file where it is defined
-exactly. Also attribute and relationship names in the JSON API
-document should match the name and casing of the attribute and
-relationship properties on the Model.
-
-For Example: given this `post` model.
+extending the `DS.Serializer` base class.
+
+
+
+A serializer in Ember Data is responsible for normalizing a payload
+from an adapter into the format that Ember Data understands. It is also
+responsible for transforming a snapshot of a record into the payload
+that an adapter will send to the backend.
+
+A serializer has two main roles in Ember Data. First it is responsible
+for taking a response from an adapter and serializing it into the
+"normalized" JSON format that Ember Data understands. Secondly, it
+transforms snapshots of records into a payload the adapter will send
+to the server when creating, updating or deleting a record.
+
+#### Ember Data's Normalized JSON Format
+
+The "normalized" JSON format that Ember Data expects is a
+[JSON API](http://jsonapi.org/) document with a couple additional
+restrictions.
+
+First, it is important to make sure that the `type` name of a record
+in the normalized JSON object exactly matches the filename of the
+model defined for this record type. By convention Model names are
+singular in Ember Data, however, the example type names shown in the
+[JSON API spec](http://jsonapi.org/format/) are pluralized. The JSON
+API spec itself is agnostic about inflection rules, however, Ember
+Data's own `JSONAPISerializer` assumes types are plural and it will
+automatically singularize the types.
```

Comment:
The spell checker is unhappy with the word "singularize" here. I'm unsure of what I should change here to make the travis build pass.